src/components/comment_list/comment_timeline.tsx

```diff
@@ -9,41 +9,51 @@
 import React, { FunctionComponent, ReactNode } from 'react';
 import { CommonProps } from '../common';
 import { EuiAvatar, EuiAvatarProps } from '../avatar';
-import { EuiCommentEventProps } from './comment_event';
 
 export interface EuiCommentTimelineProps extends CommonProps {
   /**
-   * Main icon that accompanies the comment. Should indicate who is the author of the comment. To customize, pass a `string` as an `EuiIcon['type']` or any `ReactNode`. If no icon is provided, it  will default to the `username`'s initials.
+   * Main avatar that accompanies the comment. Should indicate who is the author of the comment.
+   * Any `ReactNode`, but preferably `EuiAvatar`, or a `string` as an `EuiIcon['type']`.
+   * If no `timelineIcon` is passed, the `userAvatar` icon will be used as the avatar.
    */
   timelineIcon?: ReactNode | EuiAvatarProps['iconType'];
 
-  username: EuiCommentEventProps['username'];
+  /**
+   * Specify an `aria-label` for the `timelineIcon` when passed as an `IconType` or when nothing is passed.
+   * If no `aria-label` is passed we assume the icon is purely decorative.
+   */
+  timelineIconAriaLabel?: string;
```

Comment:
[also not a blocking comment, just my 2c] Since this also adds a `title` tooltip in addition to an aria-label, I wonder if we should make this prop more generic. Something like `timelineIconTitle` instead? Or maybe just document it:
```suggestion
   * Specify an `aria-label` and `title` for the `timelineIcon` when passed as an `IconType` or when nothing is passed.
   * If no `aria-label` is passed we assume the icon is purely decorative.
   */
  timelineIconAriaLabel?: string;
```